presto-docs/src/main/sphinx/admin/function-namespace-managers.rst

```diff
@@ -0,0 +1,78 @@
+===========================
+Function Namespace Managers
+===========================
+
+.. warning::
+
+    This is an experimental feature being actively developed. The way
+    Function Namespace Managers are configured might be changed.
+
+Function namespace managers support storing and retrieving user-defined
+functions with a persistent storage, allowing users to create, alter,
+and delete functions, as well as to reference them within queries.
+
+Function Namespace
+------------------
+
+A function namespace is a catalog-schema name (i.e. ``example.test``)
+in which each function can be uniquely identified by its name and
+parameter type list.
+
+A function namespace, such as ``example.test``, is not a full-fledged
```

Comment:
I had to read this a couple times to clarify. Maybe say that: `a function namespace can be thought of as a schema for storing functions. However, it is not a full fledged schema as it does not support...`.